src/index.ts

```diff
@@ -1,117 +1,129 @@
-import * as Promise from 'bluebird'
-import * as _ from 'lodash'
-import * as tar from 'tar-stream'
+import * as Promise from 'bluebird';
+import * as _ from 'lodash';
+import * as tar from 'tar-stream';
 
-import Bundle from './bundle'
-import { FileInfo } from './fileInfo'
-import { Resolver } from './resolver'
-import * as Utils from './utils'
+import Bundle from './bundle';
+import { FileInfo } from './fileInfo';
+import { Resolver } from './resolver';
+import * as Utils from './utils';
 
 // Import some default resolvers
-import DockerfileResolver from './resolvers/dockerfile'
-import DockerfileTemplateResolver from './resolvers/dockerfileTemplate'
-import ArchDockerfileResolver from './resolvers/archDockerfile'
-import NodeResolver from './resolvers/nodeResolver'
+import DockerfileResolver from './resolvers/dockerfile';
+import DockerfileTemplateResolver from './resolvers/dockerfileTemplate';
+import ArchDockerfileResolver from './resolvers/archDockerfile';
+import NodeResolver from './resolvers/nodeResolver';
 
 // re-export
-export { ArchDockerfileResolver,
-         Bundle,
-         DockerfileResolver,
-         DockerfileTemplateResolver,
-         FileInfo,
-         Resolver }
+export {
+	ArchDockerfileResolver,
+	Bundle,
+	DockerfileResolver,
+	DockerfileTemplateResolver,
+	FileInfo,
+	Resolver
+};
 
 export interface ResolvedBundle {
-	projectType: string,
-	tarStream: tar.Pack
+	projectType: string;
+	tarStream: tar.Pack;
 }
 
-export function resolveBundle(bundle: Bundle, resolvers: Resolver[]): Promise<ResolvedBundle> {
-	return new Promise<ResolvedBundle>((resolve: (resolved: ResolvedBundle) => void,
-                                      reject: (err: Error) => void) => {
-		const extract = tar.extract()
-		const pack = tar.pack()
-
-		extract.on('entry', (header: tar.TarHeader, stream: NodeJS.ReadableStream, next: () => void) => {
-			// Read the contents into a buffer
-			Utils.streamToBuffer(stream)
-			.then((buffer: Buffer) => {
-				// send the file along to the next tar stream regardless
-				pack.entry(header, buffer)
-
-				// create a FileInfo from the header
-				const info: FileInfo = {
-					name: Utils.normalizeTarEntry(header.name),
-					size: header.size,
-					contents: buffer
+export function resolveBundle(
+	bundle: Bundle,
+	resolvers: Resolver[]
+): Promise<ResolvedBundle> {
+	return new Promise<
+		ResolvedBundle
```

Comment:
I don't find any configuration which allows this to be changed. It shouldn't really be split as the whole line would be under the 80 chars which is the default. I'm going to do more investigation, and if I don't find any solution with the configuration I'll open an issue on the prettier side.